lib/register.php

```diff
@@ -517,3 +517,133 @@ function gutenberg_revisions_restore( $revisions_data ) {
 	return $revisions_data;
 }
 add_filter( 'wp_prepare_revision_for_js', 'gutenberg_revisions_restore' );
+
+/**
+ * Generates and caches HTML for a URL that contains OpenGraph metadata.
+ *
+ * @param string $url The URL to generate HTML for.
+ * @return mixed OpenGraph HTML, or false if there was no OpenGraph data.
+ */
+function gutenberg_oembed_opengraph( $url ) {
+	$cache_key = 'gutenberg_opengraph_' . md5( $url );
```

Comment:
Is it common to use `md5` for stuff like this? Can it just be the escaped URL?

Reply:
This is how the transient keys for embeds are generated, bit of a cargo cult copy-and-paste here I'm afraid, because I figured "well, that's how WP does it".